packer.py

```diff
@@ -48,24 +52,26 @@ def create_bins(bins: dict) -> list:
             bins[bin]['max_weight']))
     return bins_list
 
+
 def refresh_items(unfitted_items, packer):
     for item in unfitted_items:
         packer.add_item(item)
 
+
 def refresh_bin_types(bin_types, packer):
```

Comment:
Same here about naming.

Reply:
Same as above. Also I just realised that the order of parameters on both `refresh...` functions are the other way around from the `add...` methods of Packer. Think it's worth keeping consistent.